### ISSUE TITLE
Demographic and clinical variables to collect

### ISSUE THREAD

**Issue by kennethaweberii** [sct-pipeline/sci-lesion-segmentation#2]
@andrewsm8 and I are proposing that we collect the following demographic and clinical variables based on the **NINDS common data elements** (https://www.commondataelements.ninds.nih.gov/sites/nindscde/files/Doc/SCI/CDEStartupResource_SCI.pdf) and the **SCI model systems data dictionary** (https://www.nscisc.uab.edu/Public_Pages/Database_files/SCIMS%20Data%20Collection%20Forms%202016-2021.pdf):

Participant ID
Site
Ethnicity
Race
Sex
Age at time of injury
Height at time of injury
Weight at time of injury

Spinal cord injury etiology type
Spinal cord injury non traumatic etiology type

ISNCSCI days since injury (Date of ISNCSCI - Date of Injury)
ISNCSCI Neurological Level of Injury
ISNCSCI Motor Subscore Upper Extremity Right 
ISNCSCI Motor Subscore Upper Extremity Left
ISNCSCI Motor Subscore Lower Extremity Right
ISNCSCI Motor Subscore Lower Extremity Left
ISNCSCI Sensory Subscore Light Touch Right
ISNCSCI Sensory Subscore Light Touch Left
ISNCSCI Sensory Subscore Pin Prick Right 
ISNCSCI Sensory Subscore Pin Prick Left
ISNCSCI AIS Grade

Imaging days since injury (Date of imaging - Date of Injury)
Imaging metadata

**Comment by jcohenadad** [comment]
That makes sense. Maybe you can provide an example `participants.tsv` for one participant so we can agree on how data are encoded. We can also update our BIDS validator. Tagging @alexfoias

**Comment by kennethaweberii** [comment]
Here is an example participants.txt file (tab-delimited) for one participant. I was not able to upload a file with .tsv extension. We did not include any imaging parameters in this example.

[participants.txt](https://github.com/sct-pipeline/sci-lesion-segmentation/files/6696481/participants.txt)

**Comment by jcohenadad** [comment]
> Here is an example participants.txt file (tab-delimited) for one participant. I was not able to upload a file with .tsv extension. We did not include any imaging parameters in this example.
> 
> [participants.txt](https://github.com/sct-pipeline/sci-lesion-segmentation/files/6696481/participants.txt)

Looks good to me! I would suggest to add units to prevent imperial/metric confusion (we will have non-US collaborators):
- `height_at_time_of_injury_inch`
- `weight_at_time_of_injury_lbs`

**Comment by jcohenadad** [comment]
@kennethaweberii as we are about to release the first segmentation model, we would like to start doing correlations between clinical scores and lesion size. Would it be possible to add the clinical scores in the database?

**Comment by kennethaweberii** [comment]
> @kennethaweberii as we are about to release the first segmentation model, we would like to start doing correlations between clinical scores and lesion size. Would it be possible to add the clinical scores in the database?

@andrewsm8 What clinical scores can we add to the database?

**Comment by andrewsm8** [comment]
@kennethaweberii @jcohenadad, my thought would be to add ISNCSCI lower extremity motor scores, pinprick scores, and light touch scores at the time of discharge from their rehabilitation stay. How does that sound?

**Comment by jcohenadad** [comment]
I’ll let @valosekj comment on this as he is
in charge of computing correlations with clinical scores across sites

**Comment by valosekj** [comment]
> my thought would be to add ISNCSCI lower extremity motor scores, pinprick scores, and light > touch scores at the time of discharge from their rehabilitation stay. How does that sound?

This sounds good! 

AIS grade conversion at discharge would also be great if available. And potentially also scores at the time of admission (overall AIS grade, ASIA motor score (AMS), and ASIA sensory score (ASS)) if available.

Also, the demographic variables mentioned in the [first comment](https://github.com/ivadomed/model_seg_sci/issues/2#issue-910626354) would be highly appreciated:

> Participant ID
> Site
> Ethnicity
> Race
> Sex
> Age at time of injury
> Height at time of injury
> Weight at time of injury
>
> Spinal cord injury etiology type
> Spinal cord injury non traumatic etiology type

**Comment by valosekj** [comment]
After a discussion with Maxime Bouthillier (@MaximeB-radio), the following scores would also be useful, if available:
- Glasgow Coma Scale (GCS) Score at admission
- Length of stay in the ICU (in days)
- Length of Hospital Stay (in days)
- The Timing of the spine surgery (e.g. 36 hours after admission, etc.)

**Comment by dpfyffer** [comment]
> @kennethaweberii @jcohenadad, my thought would be to add ISNCSCI lower extremity motor scores, pinprick scores, and light touch scores at the time of discharge from their rehabilitation stay. How does that sound?

@andrewsm8 that would be around 3m after SCI, right? Just wanted to confirm before we identify a similar timepoint for Zurich.

**Comment by andrewsm8** [comment]
> > @kennethaweberii @jcohenadad, my thought would be to add ISNCSCI lower extremity motor scores, pinprick scores, and light touch scores at the time of discharge from their rehabilitation stay. How does that sound?
> 
> @andrewsm8 that would be around 3m after SCI, right? Just wanted to confirm before we identify a similar timepoint for Zurich.

Yes @dpfyffer, our average time from injury to rehab hospital discharge is 3.3 months.
